src/main/java/games/strategy/engine/data/GameParser.java

```diff
@@ -187,13 +188,13 @@ private void parseMinimumEngineVersionNumber(final Node minimumVersion) throws E
     if (minimumVersion == null) {
       return;
     }
-    final Version mapCompatibleWithTripleaVersion =
-        new Version(((Element) minimumVersion).getAttribute("minimumVersion"));
-    if (mapCompatibleWithTripleaVersion.isGreaterThan(ClientContext.engineVersion(), true)) {
+    final Version mapMinimumEngineVersion = new Version(((Element) minimumVersion).getAttribute("minimumVersion"));
+    if (!GameEngineVersion.of(ClientContext.engineVersion())
+        .isCompatibleWithMapMinimumEngineVersion(mapMinimumEngineVersion)) {
```

Comment:
I'm wondering what the history is on why we ignore the micro version when checking engine-map compatibility since it's a _minimum_ version check, not an _absolute_ version check.  Why do we not simply do a straight-up comparison:

```java
if (ClientContext.engineVersion().isLessThan(mapMinimumEngineVersion)) {
  throw // ...
}
```

For example, if a map author went to the trouble of stating the minimum engine version was 1.9.0.1 (possibly because a new feature was added in that release), why do we allow the map to run in engine version 1.9.0.0?  (In practice, this probably doesn't matter much, as all the maps I've seen only specify major and minor components for the minimum engine version.)

Just wondering if we can get rid of the custom engine-map compatibility check (currently in `GameEngineVersion#isCompatibleWithMapMinimumEngineVersion()`) and rely just on vanilla `Version` methods.

Reply:
Agreeing the current system doesn't make a lot of sense, but "fixing it" doesn't solve all problems. For example (AFAIK): If a map author increments the version number because of a change that requires a newer triplea version, it will still be downloaded because the game engine can't tell for what version the map was made + there's no alternative and triplea always downloads from the latest master anyways.
I'll create a dedicated issue for this...